src/package/packageVersion.ts

```diff
@@ -231,6 +231,28 @@ export class PackageVersion {
     }
   }
 
+  public static async list(
+    connection: Connection,
+    project: SfProject,
+    options: PackageVersionListOptions
+  ): Promise<PackageVersionListResult[]> {
+    // resolve/verify packages
+    const packages = options.packages.map((pkg) => {
+      const id = getPackageIdFromAlias(pkg, project);
+
+      // validate ID
+      if (id.startsWith('0Ho')) {
```

Comment:
NIT: should we expand `validateId` to return the id if it's valid? it seems like we're starting to validate the id with `.startsWith` and then call `validateId` which seems to be doing the same job as the `if` above, might just make it more intuitive

Reply:
Yes I think ID validation logic can be improved.  That was work I started doing but put on hold because the API refactoring was changing the usecases.  It can be picked up again after we get the API more solid.